react-graphql-client-library.md

```diff
@@ -65,7 +65,7 @@ const graphQLClient = axios.create();
 export default graphQLClient;
 {{< /highlight >}}
 
-Since you may want to have greater control when creating the GraphQL client instance, for example by passing in the GraphQL API endpoint or HTTP headers, you can expose it also with a function which returns the configured GraphQL client instance.
+Since you may need greater control for creating the GraphQL client instance--passing in the passing in the GraphQL API endpoint or HTTP headers, for example--you can also expose it with a function that returns the configured GraphQL client instance.
```

Comment:
"passing in the passing in the GraphQL API endpoint or HTTP headers" -> "passing in the GraphQL API endpoint or HTTP headers"